### DISCUSSION TITLE
Folder status (open / closed) to be remembered for each user

### DISCUSSION THREAD

**Discussion by sxsws** [directus/directus#13303] [Feature Requests]
For a particular task / workflow the user may want to refer to a particular group of the bookmarks or collections. Remembering which folders / collections are expanded / closed will improve usability and facilitate a better workflow.

It follows on that a user may want a certain group of collections to be visible for like a "workspace" for a particular task which would change the menu layout for each task. I envision the user being able to choose between these tasks to change the menu layout to just those collections.

**Comment by rijkvanzanten** [comment]
I think we could save these to localstorage 🙂

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
